aspnetcore/security/authentication/windowsauth.md

```diff
@@ -169,7 +169,7 @@ The [Microsoft.AspNetCore.Authentication.Negotiate](https://www.nuget.org/packag
 Instructions for joining a Linux or macOS machine to a Windows domain are available in the [Connect Azure Data Studio to your SQL Server using Windows authentication - Kerberos](/sql/azure-data-studio/enable-kerberos?view=sql-server-2017#join-your-os-to-the-active-directory-domain-controller) article. The instructions create a machine account for the Linux machine on the domain. SPNs must be added to that machine account.
 
 > [!NOTE]
-> The Linux instructions were provided for Ubuntu 18.04, and the install list provided in the article is outdated. Replace `python-software-properties` with `python3-software-properties` if needed.
+> In the [Connect Azure Data Studio to your SQL Server using Windows authentication - Kerberos](/sql/azure-data-studio/enable-kerberos?view=sql-server-2017#join-your-os-to-the-active-directory-domain-controller) article, the Linux instructions were provided for Ubuntu 18.04. The install list provided in the article is outdated. Replace `python-software-properties` with `python3-software-properties` if needed.
```

Comment:
So confused... Those instructions don't specify an Ubuntu version. I tested them with 18.04 and they mostly worked except for this package name.

Reply:
We could generalize it to say ...

> When following the guidance in the \[Connect Azure Data Studio to your SQL Server using Windows authentication - Kerberos](/sql/azure-data-studio/enable-kerberos?view=sql-server-2017#join-your-os-to-the-active-directory-domain-controller) article, replace `python-software-properties` with `python3-software-properties` if needed.

Reply:
Fair enough.